packages/bpk-component-rtl-toggle/src/BpkRtlToggle.js

```diff
@@ -41,7 +41,6 @@ class BpkRtlToggle extends React.Component {
     };
   }
 
-
```

Comment:
thanks 😄

Reply:
Well I couldn't just leave an ~extrean~ ~exteraeneo~ ~unessessary~ not needed blank line, could I 😆

Reply:
@matthewdavidson would be proud of you!